orttraining/orttraining/training_ops/cuda/cuda_training_kernels.cc

```diff
@@ -185,6 +185,8 @@ class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, Adas
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, RecordEvent);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, WaitEvent);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, YieldOp);
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, PythonOp);
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kMSDomain, 1, PythonOpGrad);
```

Comment:
There is a CPU implementation previously, we remove that in case of maintenance efforts. I would suggest to add it later if we feel there is a need for CPU run. Any thoughts what scenario that would serve?

Reply:
yeah I would prefer to defer this change in later PR (because this is one is too big for a review :)) I guess this should not be blocker for now.

Reply:
I am considering to do the CPU EP support, which would be helpful to debug the reference issue. Because sync mode would be simpler to identify some basic issues.

Reply:
Added CPU EP.